test/server_test.lua

```diff
@@ -51,6 +51,37 @@ g.test_start_stop = function()
     t.assert_equals(s.args, orig_args)
 end
 
+g.test_restart = function()
+--     Start server. Restart with same args. Check it was started. Check args
```

Comment:
Let's split this comment into several little ones and place them near corresponding sections.